config/zfs-build.m4

```diff
@@ -334,6 +334,16 @@ AC_DEFUN([ZFS_AC_DEFAULT_PACKAGE], [
 		AC_MSG_RESULT([no])
 	fi
 	AC_SUBST(DEFINE_INITRAMFS)
+
+	AC_MSG_CHECKING([whether dracut is available])
+	if test -d /usr/lib/dracut ; then
+		DEFINE_DRACUT='--define "_dracut 1"'
```

Comment:
1. Is `/usr/share/dracut` no longer supported?
2. Is the check to be performed by the host building the package?  Doesn't this break building an F23 package in an F22 host, or building the package in a host that has no Dracut?  Or are we all already deep into the Mock ideology that this is a lost cause?

Just wondering.